### PR TITLE
Adding a file about hunter

### LINKED ISSUE #18: Create a new file for lsigle hometown
## Add file for Hunter KS
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [x] Have a discussion about the pull request
- [x] Merge our pull request

### PR DESCRIPTION
**Pull request by lsigle**
Which happens to be @lsigle's hometown.

This Pull Request closes #18

### CODE REVIEW

File: places/hunter.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Hunter, KS
+
+### Great Places to Eat
+
+### Fun things to do
```
Comment (pbennyb) [inline places/hunter.md:5]: Is there anything fun to do in Hunter? I've heard it's only OK 😢

Comment (lsigle) [inline places/hunter.md:5]: > Is there anything fun to do in Hunter? I've heard it's only OK :cry:

@pbennyb There is a small cafe here, that's about it. Not really a destination for anything except hunting.

Comment (pbennyb) [inline places/hunter.md:5]: Hunting? Fitting 😄

**Comment by pbennyb** [comment]
##MERGE!##

File: places/hunter.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Hunter, KS
+
+### Great Places to Eat
+
```
Comment (lsigle) [inline places/hunter.md:4]: Local Cafe

File: places/hunter.md (hunk 1 above)
Comment (lsigle) [inline places/hunter.md:5]: - Hunting
- Fishing at Waconda Lake
- Recreation at Wilson Lake

**Pull request merged**